### PR TITLE
Int 19

### PR DESCRIPTION
**Pull request by andher1802**
(insert scope and description here)

Items:
* [x ] Ran test & live-tests
* [ x] Implemented (new) unit tests
* [ x] Removed credentials
* [ ] Removed argument pointing to staging
* [ ] Updated [documentation](sdk.up42.com)

For release:
* [ ] Bumped version
* [ ] Added changelog
* [ ] Updated announcement banner

### CODE REVIEW

File: up42/workspace.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        response_json = self.auth._request(request_type="GET", url=url)
+        workspaces_json = response_json["data"]
+        logger.info(f"Got {len(workspaces_json)} workspaces in your account")
+        return workspaces_json
+
+    def get_workspace_envs(self):
+        """
+        Returns all the environments within the selected workflow.
+        """
+        url = f"{self.auth._endpoint()}/workspaces/{self.workspace_id}/environments"
```
Comment (andher1802) [inline up42/workspace.py:41]: Add the None case

File: up42/auth.py [hunk 2]
```diff
@@ -305,3 +306,61 @@ def _request(
...
+
+        self.username = username
+        self.user_password = user_password
+        try:
+            self.env: str = kwargs["env"]
+        except KeyError:
+            self.env = "com"
+        self._get_token()
+
+    def _endpoint(self) -> str:
```
Comment (andher1802) [inline up42/auth.py:331]: I don't think this is needed anymore since it is an inherit class from auth.

Comment (andher1802) [reply up42/auth.py:331]: done! removed the _endpoint method

**Comment by chrieke** [comment]
relies on acc auth feature

**Pull request closed without merging**